AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/AppCenterLoader.java

```diff
@@ -33,6 +33,8 @@
     private static final String USE_CUSTOM_LOG_URL_KEY = "appcenter_use_custom_log_url";
     private static final String MAX_STORAGE_SIZE = "appcenter_max_storage_size";
     private static final String INITIAL_LOG_LEVEL_KEY = "appcenter_initial_log_level";
+    private static final String INITIAL_UPDATE_TRACK_KEY = "appcenter_initial_update_track";
+    private static final String INITIAL_WHEN_UPDATE_TRACK_KEY = "appcenter_initial_when_update_track";
```

Comment:
While this was fine for an internal test app UI, this is too confusing at SDK settings level. We agreed to combine those into a 3 option drop down, where the default value wouldn't call setUpdateTrack before start. You can find the poll for the third value in our team channel which should have votes by the time you see this tomorrow.

Reply:
So we will only allow before start. So name it only UPDATE_TRACK_KEY (and remove initial from value), and remove the timing.

Reply:
Fixed